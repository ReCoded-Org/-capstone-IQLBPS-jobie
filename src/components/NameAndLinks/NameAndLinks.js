import React from 'react';

const NameAndLinks = () => {
  return (
    <div className="">
      <div className="  flex flex-row pt-6  pl-44 ">
        <div className="bg-black  pt-6 p-10  " >
            <img  src='https://i.pinimg.com/236x/44/ab/21/44ab21be0821eed03d0943dd5662def8.jpg' className='w-20 bg-blue-100' alt=''/>
        </div>
      <div className=" flex flex-col   pt-6  ">
      <button
          type="button"
          className="bg-dark-gray hover:bg-[#226086]  ml-8  mb-4 w-40 text-white font-bold py-2 px-4 rounded-full  "
        >
          UPload Photo
        </button>
        <button
          type="button"
          className="bg-transparent hover:bg-dark-gray w-40  ml-8 mb-4  text-[#154360] font-semibold hover:text-white py-2 px-4 border border-dark-gray hover:border-transparent rounded-full"
        >
          Delete
        </button>
      </div>
      </div>
      <div className="pt-6 pl-44">
        <form>
        {/* <label className="mb-2 text-sm font-medium text-gray-900 sr-only dark:text-gray-300">Your Email</label> */}
          <div className="relative">
            <div className="flex absolute  items-center ">
                
              <input
                className=" p-6 pl-60  pr-60  rounded-lg border border-gray-300 "
                placeholder="No File Selected"
                required
              />
              <button
                type="button"
                className="text-white absolute left-8 rounded-full bg-dark-gray  w-32 hover:bg-[#226086] font-bold  px-4 py-2 "
              >
                Choose
              </button>
            </div>
          </div>
        </form>
      </div>
    </div>
  );
};

export default NameAndLinks;
