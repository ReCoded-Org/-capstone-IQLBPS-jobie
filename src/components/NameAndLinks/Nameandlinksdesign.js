import React from "react";
import { AiFillGithub, AiFillLinkedin, AiFillFacebook } from "react-icons/ai";

const NameAndLinksdesign = () => {
  return (
    <div className="max-w-5xl mt-16 mb-24 px-4 flex flex-col gap-4 flex-wrap mx-auto">
      <div className="flex gap-4 items-start">
        <div className="text-8xl font-bold pt-4 px-8 pb-6 cursor-pointer bg-blue-100 rounded-xl text-slate-900">
          A
        </div>
        <div className="flex flex-col items-start gap-4 pt-2">
          <h4 className="text-4xl font-bold text-slate-900">Allan Saleh</h4>
          <div className="flex flex-wrap items-center gap-2">
            <button
              type="button"
              className="bg-dark-gray hover:bg-[#226086] text-white font-bold py-1 px-10 rounded-full"
            >
              Hire
            </button>
            <button
              type="button"
              className="bg-transparent hover:bg-dark-gray text-[#154360] font-semibold hover:text-white py-1 px-10 border border-dark-gray hover:border-transparent rounded-full"
            >
              Download CV
            </button>
          </div>
        </div>
      </div>
      <div className="flex justify-between items-center flex-wrap">
        <a
          href="https://linkedin.com"
          className="flex items-center gap-1 cursor-pointer py-1 px-2 rounded-md hover:bg-gray-200"
        >
          <AiFillLinkedin className="text-2xl text-[#0072b1]" />
          Linkedin
        </a>
        <a
          href="https://github.com"
          className="flex items-center gap-1 cursor-pointer py-1 px-2 rounded-md hover:bg-gray-200"
        >
          <AiFillGithub className="text-2xl text-black" />
          GitHub
        </a>
        <a
          href="https://facebook.com"
          className="flex items-center gap-1 py-1 px-2 rounded-md hover:bg-gray-200"
        >
          <AiFillFacebook className="text-2xl text-[#3b5998]" />
          Facebook
        </a>
      </div>
    </div>
  );
};

export default NameAndLinksdesign;