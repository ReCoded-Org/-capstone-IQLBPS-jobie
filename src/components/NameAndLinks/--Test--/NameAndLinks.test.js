import renderer from 'react-test-renderer';
import NameAndLinks from '../NameAndLinks';
import React from 'react';

it('renders  NameAndLinks component correctly', () => {
  const tree = renderer.create(<NameAndLinks />).toJSON();
  expect(tree).toMatchSnapshot();
});
