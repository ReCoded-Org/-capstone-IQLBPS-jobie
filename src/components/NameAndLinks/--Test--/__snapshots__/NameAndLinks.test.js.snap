// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders  NameAndLinks component correctly 1`] = `
<div
  className=""
>
  <div
    className="  flex flex-row pt-6  pl-44  small:pl-10 "
  >
    <div
      className="bg-black  pt-6 p-10  "
    >
      <img
        alt=""
        className="w-20 bg-blue-100"
        src="https://i.pinimg.com/236x/44/ab/21/44ab21be0821eed03d0943dd5662def8.jpg"
      />
    </div>
    <div
      className=" flex flex-col   pt-6  "
    >
      <button
        className="bg-dark-gray hover:bg-[#226086]  ml-8  mb-4 w-40 text-white font-bold py-2 px-4 rounded-full  "
        type="button"
      >
        UPload Photo
      </button>
      <button
        className="bg-transparent hover:bg-dark-gray w-40  ml-8 mb-4  text-[#154360] font-semibold hover:text-white py-2 px-4 border border-dark-gray hover:border-transparent rounded-full"
        type="button"
      >
        Delete
      </button>
    </div>
  </div>
  <div
    className="pt-32  pb-32 pl-44 small:pl-10 "
  >
    <h1
      className="mb-4 text-4xl  text-dark-gray md:text-5xl lg:text-6x  pb-7 text-left"
    >
      Resume File
    </h1>
    <form>
      <div
        className="relative"
      >
        <div
          className="flex absolute  items-center "
        >
          <input
            className="  p-6 pl-44 pr-96   small:pl-24 small:pr-8  medium:pl-44  medium:pr-24  rounded-lg border border-gray-300 "
            placeholder="No File Selected"
            required={true}
          />
          <button
            className="text-white absolute left-8 rounded-full bg-dark-gray small:w-20 small:left-3 small:px-1 w-32 hover:bg-[#226086] font-bold  px-4 py-2 "
            type="button"
          >
            Choose
          </button>
        </div>
      </div>
    </form>
  </div>
</div>
`;
